src/main/java/core/basesyntax/dao/PivotDaoImpl.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.dao;
+
+import java.util.List;
+
+public class PivotDaoImpl implements PivotDao {
+    @Override
+    public void writePivotFile(String fileName, List<String> stringList) {
+        new PivotDaoCsvImpl().writePivotFile(fileName, stringList);
+    }
+}
```

Comment:
remove this class, you don't need it, you already have one implementation of this interface